aii-ks/src/main/perl/ks.pm

```diff
@@ -73,9 +68,16 @@ use constant { KS               => "/system/aii/osinstall/ks",
                INIT_SPMA_IGN_DEPS   => "/system/aii/osinstall/ks/init_spma_ignore_deps",
            };
 
+# These attributes are not configuration options
+# profile is handled separately
+use constant CCM_CONFIG_NOOPTIONS => qw(
```

Comment:
`group_readable` should not be treated differently imho. if it fails here, it will also fail in `ncm-ccm` (and currently, an unknown group is non-fatal).
wrt the whitelist, this is the `ncm-ccm` code; i can make it explicit and factor out the generation of the `ccm.conf` from `ncm-ccm` and call it here.